src/workflow/Util.cpp

```diff
@@ -171,27 +172,10 @@ bool mergeOutputTableSummaryReports(IdfObject& existingObject, const IdfObject&
 }
 
 bool addEnergyPlusOutputRequest(Workspace& workspace, IdfObject& idfObject) {
-
-  static const std::vector<IddObjectType> allowedObjects{
-    IddObjectType::Output_Surfaces_List,
-    IddObjectType::Output_Surfaces_Drawing,
-    IddObjectType::Output_Schedules,
-    IddObjectType::Output_Constructions,
-    IddObjectType::Output_Table_TimeBins,
-    IddObjectType::Output_Table_Monthly,
-    IddObjectType::Output_Variable,
-    IddObjectType::Output_Meter,
-    IddObjectType::Output_Meter_MeterFileOnly,
-    IddObjectType::Output_Meter_Cumulative,
-    IddObjectType::Output_Meter_Cumulative_MeterFileOnly,
-    IddObjectType::Meter_Custom,
-    IddObjectType::Meter_CustomDecrement,
-    IddObjectType::EnergyManagementSystem_OutputVariable,
-  };
-
-  auto iddObjectType = idfObject.iddObject().type();
-
-  if (std::find(allowedObjects.cbegin(), allowedObjects.end(), iddObjectType) != allowedObjects.end()) {
+  const auto iddObject = idfObject.iddObject();
+  const bool is_unique = iddObject.properties().unique;
+  const auto iddObjectType = iddObject.type();
+  if (!is_unique) {
```

Comment:
If not unique, indeed just check if there is the same object via dataFieldsEqual and add it if not.